charts/connect/values.yaml

```diff
@@ -39,6 +39,7 @@ connect:
 operator:
   create: false
   autoRestart: false
+  watchallNamespaces: true
```

Comment:
Wondering if we need the extra parameter. When leaving `operator.watchNamespace` an empty list `[]`, the operator will default to watching all namespaces (https://github.com/1Password/onepassword-operator/pull/40).

Having both the `operator.watchNamespace` and the `operator.watchallNamespaces` parameters as a way to configure watching all namespaces may result in unexpected behavior. Among others, when setting `operator.watchallNamespaces` to `false`, but leaving `operator.watchNamespace` to the empty list, this will result in permission errors, as all namespaces are watched by the operator, but the cluster role is not created.

How would you feel about using the empty list as a way to denote all namespaces should be watched, same as is done in the operator? We could implement this by replacing [this check for `operator.watchallNamespaces`](https://github.com/1Password/connect-helm-charts/blob/9fe696d44c938d419d50232f83c2001c8b85184c/charts/connect/templates/clusterrolebinding.yaml#L2) with a check if `operator.watchNamespaces` is set to an empty list. What do you think about that?